manager/src/main/java/org/apache/hertzbeat/manager/scheduler/CollectorAndJobScheduler.java

```diff
@@ -92,6 +92,10 @@ public class CollectorAndJobScheduler implements CollectorScheduling, CollectJob
 
     @Override
     public void collectorGoOnline(String identity, CollectorInfo collectorInfo) {
+        if (identity == null || collectorInfo == null) {
+            log.error("identity or collectorInfo can not be null if collector not existed");
+            return;
+        }
```

Comment:
if collectorInfo can not be null, the `this.manageServer.getCollectorAndJobScheduler().collectorGoOnline(identity, null);` never work.